src/core/or/circuitpadding.c

```diff
@@ -241,6 +241,20 @@ circpad_circuit_should_be_marked_for_close(circuit_t *circ, int reason)
       log_info(LD_GENERAL, "Circuit %d is not marked for close because of a "
                " pending padding machine.", CIRCUIT_IS_ORIGIN(circ) ?
                TO_ORIGIN_CIRCUIT(circ)->global_identifier : 0);
+
+      /* If the machine has had no network events at all within the
+       * last circpad_delay_t timespan, it's in some deadlock state.
+       * It's Ok to close now. */
+      if (circ->padding_info[i]->last_cell_time_sec +
+         (CIRCPAD_DELAY_INFINITE/CIRCPAD_DELAY_UNITS_PER_SECOND)+1
```

Comment:
I think we can simplify this a bit. Instead of defining the constant number `CIRCPAD_DELAY_UNITS_PER_SECOND` and the constant expression `(CIRCPAD_DELAY_INFINITE/CIRCPAD_DELAY_UNITS_PER_SECOND)+1`, instead let's define a single constant number with the result of the above expression and document exactly what it means.

Reply:
769dec7004ed28e6c8c24477b3dd18c1032b1259